tests/tests_app_examples/test_commands_and_api.py

```diff
@@ -21,32 +23,47 @@ def test_commands_and_api_example_cloud() -> None:
         # 1: Collect the app_id
         app_id = admin_page.url.split("/")[-1]
 
-        # 2: Connect to the App
+        # 2: Wait for App API to be ready
+        client = LightningClient()
+        project = _get_project(client)
+        list_lightningapps = client.lightningapp_instance_service_list_lightningapp_instances(
```

Comment:
Super nice find !!!! 
Could we move this into the `run_app_in_cloud` so it does apply to all apps?